Iirlas

iirlas opened the pull request.

iirlas committed:
Revised src\main\java\com\shaneschulte\mc\clanarena\ChallengeStart.java

iirlas committed:
Replaced src\main\java\com\shaneschulte\mc\clanarena\commands\Challenge.java With ChallengeStart.java

iirlas committed:
Merge remote-tracking branch 'remotes/origin/master' into iirlas

cgsdev0 commented on src/main/java/com/shaneschulte/mc/clanarena/ClanArenaListener.java:
```diff
@@ -0,0 +1,18 @@
+
+    @EventHandler
+    public void onChallenge (ChallengeStartEvent event) {
+        if (event.isCancelled()) {
```
instead of this, you can do (ignoreCancelled = true) in the @EventHandler annotation

cgsdev0 commented on src/main/java/com/shaneschulte/mc/clanarena/ClanArenaListener.java:
```diff
@@ -0,0 +1,18 @@
+            return;
+        }
+
+        MsgUtils.broadcastMessage("Challengers: " + event.getChallengers().toString());
```
This is probably not ideal, it will announce the entire clan's roster

iirlas committed:
Merge remote-tracking branch 'remotes/origin/master' into iirlas

iirlas committed:
Merge remote-tracking branch 'remotes/origin/CommandManager' into iirlas

# Conflicts:
#	src/main/java/com/shaneschulte/mc/clanarena/ClanArena.java

iirlas committed:
Moved To src\main\java\com\shaneschulte\mc\clanarena\listeners\ClanArenaListener.java
Renamed To src\main\java\com\shaneschulte\mc\clanarena\events\OnChallengeStart.java
Revised src\main\java\com\shaneschulte\mc\clanarena\ClanArena.java
Revised src\main\java\com\shaneschulte\mc\clanarena\commands\Challenge.java

iirlas committed:
Revised src\main\java\com\shaneschulte\mc\clanarena\ClanArena.java
Revised src\main\java\com\shaneschulte\mc\clanarena\commands\Challenge.java
Revised src\main\java\com\shaneschulte\mc\clanarena\events\OnChallengeStart.java
Revised src\main\java\com\shaneschulte\mc\clanarena\listeners\ClanArenaListener.java

iirlas committed:
SHA-1: 345db320383a08c01b84266a468ffdd28ce515ba

* Merge remote-tracking branch 'remotes/origin/CommandManager' into iirlas

# Conflicts:
#	src/main/java/com/shaneschulte/mc/clanarena/ClanArena.java

iirlas committed:
Revised src\main\java\com\shaneschulte\mc\clanarena\ClanArena.java

iirlas committed:
Revised src\main\java\com\shaneschulte\mc\clanarena\commands\Challenge.java

iirlas committed:
Merge remote-tracking branch 'remotes/origin/master' into iirlas

# Conflicts:
#	src/main/java/com/shaneschulte/mc/clanarena/ChallengeStart.java
#	src/main/java/com/shaneschulte/mc/clanarena/ClanArena.java
#	src/main/java/com/shaneschulte/mc/clanarena/commands/ChallengeCmd.java

iirlas committed:
Re-added src\main\java\com\shaneschulte\mc\clanarena\commands\Challenge.java
